src/components/ObsDetails/ObsDetails.js

```diff
@@ -159,6 +165,25 @@ const ObsDetails = (): Node => {
     } );
   };
 
+  const createIdentificationMutation = useAuthenticatedMutation(
+    ( idParams, optsWithAuth ) => createIdentification( idParams, optsWithAuth ),
+    {
+      onSuccess: data => setIds( [...ids, data[0]] ),
+      onError: e => {
+        let error = null;
+        if ( e ) {
+          error = t( "Couldnt-create-identification", { error: e.message } );
+        } else {
+          error = t( "Couldnt-create-identification", { error: t( "Unknown-error" ) } );
```

Comment:
If possible, try to avoid using translation strings with variables as they are almost always harder to translate. Something like `t( "Couldnt-create-identification-unknown-error" )`. It will feel a bit repetitive, but it keeps translators happy.

Reply:
Would `t( "Couldnt-create-identification", { error: e.message } );` be replaced with just `t( "Couldnt-create-identification")`? Or do we want strings for each error message?